### ISSUE TITLE
Notification issue

### ISSUE THREAD

**Issue by edwood1981** [balazsbanto/ticket-buyer-bot#5]
Hello,

First, thanks a lot for your work!

I have an issue with the APK. On my phone,  I opened the accès to notifications and accessibility, the language is in english. The bot works to add to cart. But it doesn't work for the notifications. I mean the bot doesn't open the app (even if the app is yet open).

Any idea for a solution?

Thanks

**Comment by shellduck8** [comment]
Same issue here
> Hello,
> 
> First, thanks a lot for your work!
> 
> I have an issue with the APK. On my phone,  I opened the accès to notifications and accessibility, the language is in english. The bot works to add to cart. But it doesn't work for the notifications. I mean the bot doesn't open the app (even if the app is yet open).
> 
> Any idea for a solution?
> 
> Thanks

**Comment by balazsbanto** [comment]
The app no longer works, TicketSwap has released a patch, sorry.
